ApplyToTheIndustry/Assets/Scripts/GameResources/Action.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Action : MonoBehaviour
+{
+    public TimeManager tm;
```

Comment:
Rather than having references to these systems, I would manage it so they call a singleton - such as the service locator, when they need to. This way we can decouple these two. For now though, it is fine.